pages/demos/app-bar.js

```diff
@@ -4,7 +4,11 @@ import React from 'react';
 import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
 
 const req = require.context('docs/src/pages/demos/app-bar', false, /\.md|\.js$/);
-const reqSource = require.context('!raw-loader!../../docs/src/pages/demos/app-bar', false, /\.js$/);
+const reqSource = require.context(
+  '!raw-loader!../../docs/src/pages/demos/app-bar',
+  false,
+  /\.(js|tsx)$/,
```

Comment:
Will we be able to run the TypeScript version at some point? Looking at the imports, it seems we are only using the `.tsx` file for displaying the source of the demo.